File: mobi_client/mobi_client/mobi_hotfix/mobi_hotfix/HFScripts/HFConfig/ConfigModel/ActTemplateConfigModel.cs
public class ActTemplateConfigModel : ConfigSystem.ConfigModel {
    public int actId;//活动id
    public int isEnable;//是否开启
    public string desc;//描述
    public string name;//活动名字
    public int type;//类型
    public string startTime;//开启时间
    public string endTime;//关闭时间
    public string rewardTime;//领奖时间
    public System.Collections.Generic.List<int> argsInt;//额外参数
    public System.Collections.Generic.List<string> argsString;//额外参数2
}

File: mobi_client/mobi_client/mobi_config/excel_check/temporary/java/ActTemplateConfigModel.java
package com.mobi.config.configModel;
import com.mobi.config.ConfigModel;
import com.mobi.config.wrappedArrayList.*;
public class ActTemplateConfigModel extends ConfigModel {
	public int actId;//活动id
	public int isEnable;//是否开启
	public String desc;//描述
	public String name;//活动名字
	public int type;//类型
	public String startTime;//开启时间
	public String endTime;//关闭时间
	public String rewardTime;//领奖时间
	public ArrInt argsInt;//额外参数
	public ArrString argsString;//额外参数2
}